docker/README.md

```diff
@@ -0,0 +1,60 @@
+README
+======
+
+This folder contains a Dockerfile and some utilities that simplify the creation of a Docker container for Personfinder development and testing.
+
+Steps to create the container
+-----------------------------
+
+1. Setup the environment for Docker (see Docker's webpage)
+2. Build Person Finder Docker image
+
+    From `personfinder/docker` folder, run
+
+        $ docker build -t <image-name> .
+
+    e.g.
+
+        $ docker build -t personfinder-image .
+
+    This will take a while and will create the docker image.
+
+3. Create the container
+
+    Run the following command
+
+    $ ./run-container.sh <image-name> [<container-name>]
+
+    e.g.
+
+        $ ./run-container.sh personfinder-image personfinder-container  # Container's name is optional
+
+    This will create and run at the same time the container image. To manage the container's lifecycle (start,
+    stop, destroy, etc.) please refer to the Docker's documentation.
+
+4. Run the Person Finder local server and test it from host's web browser
+
+    In the SHELL created inside the Person Finder container, run the following command to run the
+    Person Finder local server:
+
+        # gae-run-app.sh
+
+    You will be able to access the server from the Host's web browser at https://<container-ip-address>:8000
+
+
+5. Initialize the datastore:
+
+    open one more SHELL inside the running container with the following command (Docker version 1.3.0 or higher,
+    for older Docker versions, docker attach command should do the job as well).
+
+        # docker exec -it <container-name> /bin/bash
```

Comment:
Oh I see. You should use "$" instead of "#" for this line because it's a command running on the host. Also, it would be clearer if you say:

> open one more SHELL inside the running container by running the following command on the host's shell